GitCommands/Settings/ConfigFileSettings.cs

```diff
@@ -136,7 +136,7 @@ public Encoding FilesEncoding
 
         private Encoding? GetEncoding(string settingName)
         {
-            string encodingName = GetValue(settingName);
+            string encodingName = GetValue(settingName).ToLower();
```

Comment:
Could you please elaborate how this is addressing the issue?

Reply:
I wrote it in the commit message.

Suppose the setting is UTF-8. Here it is converted to lower case, so `encodingName` is `utf-8`. This value exists in `AppSettings.AvailableEncodings`, and it is mapped to BOM-less UTF-8.

Reply:
Thank you for the clarification. Please add a comment to the code explaining this, so that this change won't get undone by accident.

Reply:
Done